src/go/k8s/tests/e2e/existing-cluster/01-assert.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: existing-cluster
+status:
+  readyReplicas: 2
```

Comment:
so it is just testing that the cluster with the annotation starts? If yes, can't we just add the annotation to one of the existing clusters?

Reply:
Right! `readyReplicas` was also not present in v22.1